srushteeMarathe.txt

```diff
@@ -0,0 +1,3 @@
+My name is Srushtee Marathe
+I am a programmer and a coding instructor.
+I love to work with React Native
```

Comment:
Why do you love to work with React Native?

Reply:
It has lots of components to work on

Reply:
Amazing 💯